source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -110,6 +110,30 @@ namespace PlayFab
             auto playFabEmitEventResponse = std::make_shared<PlayFabEmitEventResponse>();
             playFabEmitEventResponse->emitEventResult = emitResult;
 
+            std::shared_ptr<PlayFabError> emitEventError  = std::make_shared<PlayFabError>();
+
+            emitEventError->ErrorName = "PlayFabEventPipeline IntakeEvent Error";
+            emitEventError->ErrorMessage = "PlayFabEventPipeline did not accept the event. Please see ErrorDetails for more information.";
+            emitEventError->ErrorCode = PlayFabErrorCode::PlayFabErrorEventNotFound;
```

Comment:
>PlayFabErrorEventNotFound [](start = 58, length = 25)

this seems like a misleading error code. what does it mean that the event wasn't found? found from what? that doesn't seem like an accurate reflection of the problem which was that the event couldn't be placed in the event buffer

Reply:
I can agree that this error code is misleading. 

But there really is not a good error code for this particular issue. To add that error code, we will need to make a pf-main request to add that error code to all other api's as well (like java, unity and php) which won't have an event pipeline, adding noise to our already very long error code list. Would we want to leave this errorcode blank and expect users to read the ErrorName? Would a -1 casted PlayFab error code be acceptable (or even possible if it's an enum which I thought would be an unsigned int)